lib/vanagon/driver.rb

```diff
@@ -50,6 +53,7 @@ def self.logger
       @@logger
     end
 
+
```

Comment:
![room-for-activities](https://cloud.githubusercontent.com/assets/127475/14093221/20994546-f501-11e5-8e30-967c86783713.jpg)